http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutesJsonCodecSpec.scala

```diff
@@ -67,7 +67,24 @@ class LeoRoutesJsonCodecSpec extends FlatSpec with Matchers {
       json <- io.circe.parser.parse(inputString)
       r <- json.as[RuntimeConfigRequest.DataprocConfig]
     } yield r
-    decodeResult.leftMap(_.getMessage) shouldBe Left("Negative number is not allowed: DownField(masterDiskSize)")
+    decodeResult.leftMap(_.getMessage) shouldBe Left("Minimum required disk size is 50GB: DownField(masterDiskSize)")
+  }
+
+  it should "fail with minimumDiskSizeDecodingFailure when masterDiskSize is less than 50" in {
```

Comment:
this is fine too...I would probably add the test in https://github.com/DataBiosphere/leonardo/blob/develop/core/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/JsonCodecSpec.scala#L8 for just `diskSizeDecoder`

Reply:
I added `minimumDiskSizeDecodingFailure`  because we use the `negativeNumberDecodingFailure` in other places. and good point, I'll move my test now!

Reply:
@Qi77Qi I moved my unit test to `JsonCodecSpec` and have a unit test for a GCE config and a Dataproc config!